tests/headers/derive-debug-opaque-template-instantiation.hpp

```diff
@@ -0,0 +1,10 @@
+// bindgen-flags: --force-derive-debug
+
+template<typename T, int N>
+class Opaque {
```

Comment:
It's worth adding a comment that this is opaque because of the non-type template parameter, which is easy to miss.